index.html

```diff
@@ -16,25 +16,33 @@
 <!--===================================================================================-->
 
 <section id="what">
-  <h2>What can I do with it?</h2>
+  <h2>All the apps you need, all in one place.</h2>
   <ul>
-    <li class="create">
-      <h3>Create</h3>
-      <p>Create Google-Docs-like spreadsheets, documents, forms, etc. with
-        EtherCalc, Etherpad, Sandforms, Draw.io, and more.</p>
+    <li class="chat">
+      <h3>Chat</h3>
+      <p>Sync up with your colleagues securely using Rocket.Chat.</p>
     </li>
-    <li class="collaborate">
-      <h3>Collaborate</h3>
-      <p>Share documents, diagrams, and other files with your colleagues and friends, and collaborate in real-time.</p>
+    <li class="file-storage">
+      <h3>File Storage</h3>
+      <p>Store all your files in the cloud and share them with others using Davros.</p>
```

Comment:
Although we also do say "You can use Sandstorm in the cloud on a variety of hosting services". But perhaps this is different.